content/team/kristaS.md

```diff
@@ -0,0 +1,7 @@
+---
+name : Krista Springs
```

Comment:
Should be: Spriggs

Reply:
sorry about that, just made the change! @kristaxox